soc/xtensa/intel_adsp/common/boot.c

```diff
@@ -297,6 +292,7 @@ static __imr void lp_sram_init(void)
 	}
 
 	CAVS_SHIM.ldoctl = SHIM_LDOCTL_LPSRAM_LDO_BYPASS;
+	bbzero((void *)L2_SRAM_BASE, L2_SRAM_SIZE);
```

Comment:
Shouldn't these be `LP_SRAM_BASE` and `LP_SRAM_SIZE`?

Reply:
Whoops, yes.

Reply:
Fixed.  In fact that exposed the fact we had bad sizes (presumptively bad anyway; either that or there's missing code to enable sram bank somewhere) for the LP-SRAM block.  Useful catch.